server/api/orders.js

```diff
@@ -0,0 +1,37 @@
+const {Order} = require('../db/models')
+const router = require('express').Router()
+module.exports = router
+
+router.get('/', async (req, res, next) => {
+  try {
+    const orders = await Order.findAll()
+    res.json(orders)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.get('/:orderId', async (req, res, next) => {
+  try {
+    const order = await Order.findById(Number(req.params.orderId))
+    res.json(order)
+  } catch (err) {
+    next(err)
+  }
+})
+
+router.post('/', async (req, res, next) => {
```

Comment:
Since we want an 'itemized receipt' view of Order, wouldn't we need to also create an OrderProduct instance where we have each product with its quantity and price-at-time-of-order-creation?